### DISCUSSION TITLE
点开文章后无法伸缩侧边栏

### DISCUSSION THREAD

**Discussion by fanghsiu** [YunYouJun/hexo-theme-starry#348] [Q&A 疑难杂症] [answered]
```yaml
pjax: 
  enable: false
```
pjax 关闭导致点开文章后无法伸缩侧边栏，没有按钮，大佬你的网站也是这样。虽然几乎没有影响，但还是想知道有没有不开 pjax 的解决方案。 @YunYouJun

**Comment by YunYouJun** [comment, marked as answer]
能否录屏失意，不明白……

**Reply by fanghsiu** [reply]
![image](https://user-images.githubusercontent.com/79410512/191835461-a257f090-3166-489d-9f5e-b139969ece3a.png)
![image](https://user-images.githubusercontent.com/79410512/191835477-54c29a08-60ba-4f3f-bdee-f7756d53e039.png)

点开文章后图中红框处按钮没了，所以没法伸缩侧边栏，是 bug 还是就这样设计的？

**Reply by YunYouJun** [reply]
是这样设计的，当页面尺寸更小时，关闭按钮会重新出现。

首页因为 banner 居中，所以会显示关闭按钮。

**Reply by fanghsiu** [reply]
好的，谢谢解答。
